jsonschema/validator_test.go

```diff
@@ -50,8 +51,11 @@ func TestValidate(t *testing.T) {
 		}
 
 		got, err := v.Validate(raw)
-
-		if (err != nil) != test.wantErr {
+		if err != nil && test.wantErr != nil {
+			if err.Error() != test.wantErr.Error() {
+				t.Errorf("Test %q - got error %v, want error %t", test.description, err, test.wantErr)
```

Comment:
Minor nit: since you are comparing the error strings, you could output the error string in the failure message to improve readability of the test result. Your choice.

Reply:
since we are using Errorf I'm pretty sure it'll be equivalent.

Reply:
I tried it out, it was a bit difficult to read:
```
=== RUN   TestValidate
--- FAIL: TestValidate (0.01s)
    validator_test.go:56: Test "invalid image JSON" - got error invalid schema: [URL: URL is required caption: caption is required credit: credit is required crops: crops is required datePhotoTaken: datePhotoTaken is required orientation: orientation is required originalSize: originalSize is required type: type must be one of the following: "image"], want error &{%!t(string=Invalid schema: [URL: URL is required caption: caption is required credit: credit is required crops: crops is required datePhotoTaken: datePhotoTaken is required orientation: orientation is required originalSize: originalSize is required type: type must be one of the following: "image"])}
```
not a huge deal.

Reply:
basically `got error invalid schema: ...` vs `want error &{%!t(string=Invalid schema: ...`